docs/source/api-proposal.rst

```diff
@@ -1,1000 +0,0 @@
-Calico API Design
```

Comment:
Should we not leave this in place until we actually switch to etcd?

Perhaps retitle it as something like 0MQ API, and add a link upfront to the etcd doc, with a note that we're in the process of transitioning.